src/test/server/database/types/work_machine.py

```diff
@@ -47,9 +48,13 @@ def test_resources_getter1(self) -> None:
 
 class WorkMachineSerializableTest(AbstractSerializableTest):
     def setUp(self) -> None:
-        self._optional_properties = ["resources"]
+        self._optional_properties = ["resources", "ssh_config"]
```

Comment:
My understanding was that ssh_config is no longer optional?

Reply:
Yea but it had a default value of None that is why I put it there, should I remove None as default?

Reply:
If it is mandatory, then I think it should not be part of the `optional_properties` :)

Reply:
Also mandatory == no need for default value afaiu

Reply:
But that is also true for resources?

Reply:
Then I'm not sure why these are optional either?

Reply:
yea but maybe for tests, because now if I try to fix something I get 54 failed tests.

Reply:
When a property is mandatory it has to be specified when creating the object.
When a property is None, that is treated as the property not being specified.
self._optional_properties is a list of properties that are not mandatory.

Reply:
I think ssh_config should not be allowed to be None.
The from_dict method will need to be adjusted.